install_config/install/prerequisites.adoc

```diff
@@ -783,6 +789,9 @@ This will destroy any Docker containers or images currently on the host.
 # rm -rf /var/lib/docker/*
 # systemctl restart docker
 ----
++
+If there is any content in *_/var/lib/docker/_*, it has to be deleted. Files
+will be present if Docker has been used prior to the install of OpenShift.
```

Comment:
s/install/installation/

IBM Style Guide: use "install" as verb, "installation" for noun/adj.